### PR TITLE
Add small `border-radius` to content images on tablet and up

### PR DESCRIPTION
**Pull request by Volker-E**
Also automatically added are new vendor prefixes supported by newest Autoprefixer when building the CSS files.

Bug: T186482

### CODE REVIEW

File: css/wmui-style-guide.css [hunk 1]
```diff
@@ -400,7 +400,9 @@ a:hover {
 abbr[title] {
   border-bottom: none; /* 1 */
   text-decoration: underline; /* 2 */
-  text-decoration: underline dotted; /* 2 */
+  -webkit-text-decoration: underline dotted;
+     -moz-text-decoration: underline dotted;
+          text-decoration: underline dotted; /* 2 */
```
Comment (prtksxna) [inline css/wmui-style-guide.css]: Is this repeated in #62?

Comment (Volker-E) [reply css/wmui-style-guide.css]: @prtksxna Yeah, that's a problem of grunt build with autoprefixer. The new version of autoprefixer adds those vendor prefixes here.

Comment (Volker-E) [reply css/wmui-style-guide.css]: @prtksxna See #63, will update other patches accordingly after that.

Comment (prtksxna) [reply css/wmui-style-guide.css]: So #63 is closed now. Will you be rebasing?

**Pull request merged**